addons/project/models/project.py

```diff
@@ -2088,14 +2088,14 @@ def action_dependent_tasks(self):
             'res_model': 'project.task',
             'type': 'ir.actions.act_window',
             'context': {**self._context, 'default_depend_on_ids': [Command.link(self.id)]},
+            'domain': [('depend_on_ids', '=', self.id)],
```

Comment:
I don't understand this change 🤔
Perhaps I miss something. 😇

Reply:
This is supposed to fix the following point of the task: project.task: the 'blocking' stat button is missing a domain. Is there something wrong here?

Reply:
Oh okay sorry 😇